app/styles/custom/_containers.scss

```diff
@@ -2,7 +2,7 @@
   display: flex;
```

Comment:
variables for header padding

Reply:
Header padding is not defined as a variable, it is always hard coded depending on the application.